Plugins/NativeEngine/Source/IndexBuffer.cpp

```diff
@@ -1,11 +1,19 @@
 #include "IndexBuffer.h"
+#include <arcana/threading/task.h>
+#include <arcana/threading/task_schedulers.h>
 
 namespace Babylon
 {
-    IndexBuffer::IndexBuffer(gsl::span<uint8_t> bytes, uint16_t flags, bool dynamic)
+    IndexBuffer::IndexBuffer(gsl::span<uint8_t> bytes, 
```

Comment:
Since `bytes` is derived from `dataRefs` at the construction site, should this parameter be removed and `m_bytes` is calculated inside the constructor?

Reply:
Approach has changed, so this is no longer valid.